scripts/apps/extension-points/index.js

```diff
@@ -0,0 +1,25 @@
+import * as directive from './directives';
+import * as svc from './services';
+
+/*
```

Comment:
We have a documentation system (not sure how well it's working at the moment), but the syntax for it is as follows:
```js
/**
 * @ngdoc module
 * @module superdesk.apps.extension-points
 * @name superdesk.apps.extension-points
 * @packageName superdesk.apps
 * @description DESCRIPTION GOES HERE
 */
```
Please try and use the same pattern throughout, when documenting both directives and services. You can get inspired from other modules in the codebase.